resources/dax_clusters.go

```diff
@@ -0,0 +1,314 @@
+			{
+				Name:        "aws_dax_cluster_nodes",
+				Description: "Represents an individual node within a DAX cluster.",
+				Resolver:    fetchDaxClusterNodes,
+				Columns: []schema.Column{
+					{
+						Name:        "cluster_cq_id",
+						Description: "Unique CloudQuery ID of aws_dax_clusters table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "availability_zone",
+						Description: "The Availability Zone (AZ) in which the node has been deployed.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "endpoint_address",
+						Description: "The DNS hostname of the endpoint.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("Endpoint.Address"),
+					},
+					{
+						Name:        "endpoint_port",
+						Description: "The port number that applications should use to connect to the endpoint.",
+						Type:        schema.TypeInt,
+						Resolver:    schema.PathResolver("Endpoint.Port"),
+					},
+					{
+						Name:        "endpoint_url",
+						Description: "The URL that applications should use to connect to the endpoint",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("Endpoint.URL"),
+					},
+					{
+						Name:        "node_create_time",
+						Description: "The date and time (in UNIX epoch format) when the node was launched.",
+						Type:        schema.TypeTimestamp,
+					},
+					{
+						Name:        "node_id",
+						Description: "A system-generated identifier for the node.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "node_status",
+						Description: "The current status of the node",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "parameter_group_status",
+						Description: "The status of the parameter group associated with this node",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+func fetchDaxClusters(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().DAX
+
+	// DAX is only available in the following regions.
+	if _, ok := map[string]struct{}{
+		"ap-northeast-1": {},
+		"ap-southeast-1": {},
+		"ap-southeast-2": {},
+		"ap-south-1":     {},
+		"cn-northwest-1": {},
+		"cn-north-1":     {},
+		"eu-central-1":   {},
+		"eu-west-1":      {},
+		"eu-west-2":      {},
+		"eu-west-3":      {},
+		"sa-east-1":      {},
+		"us-east-1":      {},
+		"us-east-2":      {},
+		"us-west-1":      {},
+		"us-west-2":      {},
+	}[c.Region]; !ok {
+		return nil
+	}
```

Comment:
add a TODO here referencing #263, this should be automatic for all services in the account/region multiplexer